### DISCUSSION TITLE
How to remove Lightspark

### DISCUSSION THREAD

**Discussion by heatdream** [lightspark/lightspark#714] [Q&A] [answered]
Hey, I want to remove Lightspark and want to know if there is a simple mthode to do this, or an Guide for it?
does anyone have any experience with it?

**Comment by huwdp** [comment]
There are multiple ways to uninstall this. I don't know on-top of my head but there should be an uninstaller for Windows. What platform are you on?

**Reply by heatdream** [reply]
i'm working on Linux

**Comment by huwdp** [comment, marked as answer]
If you installed Lightspark via a deb, then you can run something like ```sudo apt-get remove lightspark```. If you built it from source then you can run ```xargs rm < install_manifest.txt```. See PR (https://github.com/lightspark/lightspark/pull/517)

**Reply by heatdream** [reply]
Hey thanks for That fast reply
